ci: temporary debugging output for release process

jacderida opened the pull request:
Something is causing the build agent to run out of space. I have a suspicion it's possible that it's something to do with how `release-plz` determines changes in the crates, and it could be struggling because the number of crates in our workspace has grown.

The first thing is to enable debugging output, because I seen some `debug!` usage in the `release-plz` codebase. Hoping maybe that could provide some clues if it is `release-plz` that is causing the disk usage.

If not, each step has output to display the disk usage, which hopefully may give us other clues.

This commit can be reverted when we have solved the issue.

## Description

reviewpad:summary

Yoga07 commented:
bors r+

b-zee commented on testnet/bin.rs:
```diff
@@ -46,7 +46,7 @@ const SAFE_NODE_EXECUTABLE: &str = "sn_node.exe";
 const BASE_TRACING_DIRECTIVES: &str = "testnet=info,sn_launch_tool=debug";
 const NODES_DIR: &str = "local-test-network";
-const DEFAULT_INTERVAL: &str = "10000";
+const DEFAULT_INTERVAL: &str = "100";
```
Was this changed on purpose? I know that the interval can generally be lower, but not sure if this is only reasonable on more powerful machines.

joshuef replied:
Yeh that was intentional. For me locally (and on the prior CI runs) this seems to have been working well

joshuef commented:
bors r+

jacderida committed:
ci: temporary debugging output for release process

Something is causing the build agent to run out of space. I have a suspicion it's possible that it's
something to do with how `release-plz` determines changes in the crates, and it could be struggling
because the number of crates in our workspace has grown.

The first thing is to enable debugging output, because I seen some `debug!` usage in the
`release-plz` codebase. Hoping maybe that could provide some clues if it is `release-plz` that is
causing the disk usage.

If not, each step has output to display the disk usage, which hopefully may give us other clues.

This commit can be reverted when we have solved the issue.
